src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestOpArgument.ll

```diff
@@ -19,8 +19,37 @@ entry:
   store %Qubit* %q, %Qubit** %10
   %11 = call %Callable* @__quantum__rt__callable_create([4 x void (%TupleHeader*, %TupleHeader*, %TupleHeader*)*]* @PartialApplication__2, %TupleHeader* %6)
   call void @Microsoft__Quantum__Testing__QIR__Apply__body(%Callable* %11)
+  %12 = call %TupleHeader* @__quantum__rt__tuple_create(i64 ptrtoint ({ %TupleHeader, %Callable*, %Qubit*, %Qubit* }* getelementptr ({ %TupleHeader, %Callable*, %Qubit*, %Qubit* }, { %TupleHeader, %Callable*, %Qubit*, %Qubit* }* null, i32 1) to i64))
+  %13 = bitcast %TupleHeader* %12 to { %TupleHeader, %Callable*, %Qubit*, %Qubit* }*
+  %14 = getelementptr { %TupleHeader, %Callable*, %Qubit*, %Qubit* }, { %TupleHeader, %Callable*, %Qubit*, %Qubit* }* %13, i64 0, i32 1
+  %15 = call %Callable* @__quantum__rt__callable_create([4 x void (%TupleHeader*, %TupleHeader*, %TupleHeader*)*]* @Microsoft__Quantum__Testing__QIR___Choose, %TupleHeader* null)
+  store %Callable* %15, %Callable** %14
+  %16 = getelementptr { %TupleHeader, %Callable*, %Qubit*, %Qubit* }, { %TupleHeader, %Callable*, %Qubit*, %Qubit* }* %13, i64 0, i32 2
+  store %Qubit* %q, %Qubit** %16
+  %17 = getelementptr { %TupleHeader, %Callable*, %Qubit*, %Qubit* }, { %TupleHeader, %Callable*, %Qubit*, %Qubit* }* %13, i64 0, i32 3
+  store %Qubit* %q, %Qubit** %17
+  %18 = bitcast %TupleHeader* %8 to { %TupleHeader, %Qubit*, %Qubit* }*
```

Comment:
why? %18 doesn't seem to be consumed by anybody below

Reply:
Hu, this seems wrong in any case; %8 contains the pointer to the constant array containing the functions for SWAP. It seems like there is something wrong here. I'll look into it - thanks for pointing it out!

Reply:
This is fixed now and this bitcast no longer exists (it is now part of the lifted partial application).